benchmarks/benchmarks/model_acc/bench_rgcn.py

```diff
@@ -0,0 +1,127 @@
+import dgl
+from dgl.nn.pytorch import RelGraphConv
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from .. import utils
+
+class RGCN(nn.Module):
+    def __init__(self,
+                 num_nodes,
+                 n_hidden,
+                 num_classes,
+                 num_rels,
+                 num_bases,
+                 num_hidden_layers,
+                 dropout):
+        super(RGCN, self).__init__()
+        self.layers = nn.ModuleList()
+        # i2h
+        self.layers.append(RelGraphConv(num_nodes, n_hidden, num_rels, "basis",
+                                        num_bases, activation=F.relu, dropout=dropout,
+                                        low_mem=True))
+        # h2h
+        for i in range(num_hidden_layers):
+            self.layers.append(RelGraphConv(n_hidden, n_hidden, num_rels, "basis",
+                                            num_bases, activation=F.relu, dropout=dropout,
+                                            low_mem=True))
+        # o2h
+        self.layers.append(RelGraphConv(n_hidden, num_classes, num_rels, "basis",
+                                        num_bases, activation=None, low_mem=True))
+
+    def forward(self, g, h, r, norm):
+        for layer in self.layers:
+            h = layer(g, h, r, norm)
+        return h
+
+def evaluate(model, g, feats, edge_type, edge_norm, labels, idx):
+    model.eval()
+    with torch.no_grad():
+        logits = model(g, feats, edge_type, edge_norm)
+        logits = logits[idx]
+        _, indices = torch.max(logits, dim=1)
+        correct = torch.sum(indices == labels)
+        return correct.item() * 1.0 / len(labels) * 100
+
+@utils.benchmark('acc')
+@utils.parametrize('data', ['aifb', 'mutag'])
+def track_acc(data):
+    # args
+    if data == 'aifb':
+        num_bases = -1
+        l2norm = 0.
+    elif data == 'mutag':
+        num_bases = 30
+        l2norm = 5e-4
+    elif data == 'am':
```

Comment:
am is not in the dataset.

Reply:
Right now processing am dataset takes too long. I will add them back when we have properly setup the dataset pipeline.